docs/device-and-supply-chain-modelling.md

```diff
@@ -0,0 +1,78 @@
+# Device and Supply Chain modelling
+
+## Intro
+This document sets out the accumulated knowledge and / or assumptions about:
++ the evidence available within a device which can be used for attestation
++ models for firmware release lifecycles 
++ observations on how multiple products may be released from a single source and the implications for provisioning data
+
+Why are these significant? The models here have driven the design of the Veraison system, in particular the way supply chain data is provisioned into the system and the Endorsement store is constructed to allow verification queries to access reference values to be used for evidence appraisal. Those design conclusions can be found in other documents.
+
+The Veraison project welcomes feedback from real world examples as to the accuracy of the models and assumptions used here. The project will be seeking confirmation on these models from industry partners. If you have views on these topics, please provide relevant commentary via a github issue or start a conversation on the Zulip channel.
+
+## Device evidence classification
+The evidence within an attestation report can be categorised into the following classes. These classes are not an exhaustive set of all types of evidence that may be found within a report but are those necessary for correlating evidence with endorsement / reference value queries.
+
+### Trust Anchor
+All devices are ASSUMED to contain a Trust Anchor (TA) or a descendant from a Trust Anchor. This is normally provisioned into the device at manufacturing time but may also be provisioned to the device on enrolling into a management structure. The TA takes the form of a key or a seed from which an Attestation signing key can be derived. The attestation report will contain information, referred to here as a Trust Anchor ID (TA-ID), which a Verifier can use to locate an endorsement providing information on the public value of the relevant key to use to validate the signature on a report. The Trust Anchor may have unique-to-device or group scope (i.e. multiple devices contain the same trust anchor). The latter case has been used as one solution for privacy protection use cases. 
```

Comment:
Minor comment: To improve readability consider splitting the sentence into two simple sentences:

referred to here as a Trust Anchor ID (TA-ID), which a Verifier can use to locate an endorsement. This endorsement provides information on the public part of the relevant key, 
which is used to validate the signature of a report.

Reply:
Fixed